rpcs3/Emu/RSX/GL/GLHelpers.h

```diff
@@ -31,6 +31,11 @@
 #define GL_INTERPRETER_FRAGMENT_BLOCK 7
 #define GL_COMPUTE_BUFFER_SLOT(index) (index + 8)
 
+// Keyword required only by Windows (used in log_debug)
+#ifndef APIENTRY
+#define APIENTRY
```

Comment:
This word isn't a noop, iirc it defines function calling convention. It should be defined in windows.h iirc

Reply:
Confirmed, APIENTRY is an alias for WINAPI which is an alias for __stdcall

Reply:
It does define such convention on Windows, but [Unix doesn't need this convention](https://stackoverflow.com/a/34249928), just `void log_debug` is enough, this is also done on glext.h (see: https://github.com/RPCS3/rpcs3/blob/master/3rdparty/GL/glext.h#L45) but due to massive header entanglement it doesn't get picked up on log_debug

Reply:
I agree with that, but this should be blocked from being defined this way on windows. Wrap the check in ifndef _WIN32.